src/extension/rn-extension.ts

```diff
@@ -103,7 +103,7 @@ function setupAndDispose<T extends ISetupableDisposable>(setuptableDisposable: T
 function warnWhenReactNativeVersionIsNotSupported(): void {
     return reactNativeProjectHelper.validateReactNativeVersion().done(() => { }, reason => {
         TelemetryHelper.sendSimpleEvent("unsupportedRNVersion", { rnVersion: reason });
-        const shortMessage = `React Native Tools only supports React Native versions 0.19.0 and later`;
+        const shortMessage = `React Native Tools extension only supports React Native versions 0.19.0 and later`;
```

Comment:
How about, "React Native Tools extension needs React Native v0.19.0 or later to be installed in <PROJECT_ROOT>/node_modules/" ?